metricflow/protocols/sql_client.py

```diff
@@ -175,6 +187,9 @@ class SqlEngineAttributes(Protocol):
     sql_engine_type: ClassVar[SqlEngine]
 
     # SQL Engine capabilities
+    # The isolation levels supported as options through the SqlClient API. This may be a subset of the isolation levels
+    # supported by the engine until implementation / testing is complete.
+    supported_isolation_levels: ClassVar[Sequence[SqlIsolationLevel]]
```

Comment:
OK. We'll need to do our big version bump and document this as a breaking change for developers in the release notes, but otherwise this should be fine - we'll need this information when we update the public APIs regardless.